src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final String DEFAULT_COLOR = "white";
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int FIGURE_TYPES_COUNT = 5;
+
+    public Figure getRandomFigure() {
+        int figurType = RANDOM.nextInt(FIGURE_TYPES_COUNT);
+        String color = new ColorSupplier().getRandomColor();
+        switch (figurType) {
+            case 0:
+                return new Circle(color, getRandomSize());
+            case 1:
+                return new Square(color, getRandomSize());
+            case 2:
+                return new Rectangle(color, getRandomSize(), getRandomSize());
+            case 3:
+                return new RightTriangle(color, getRandomSize(),
+                        getRandomSize(), getRandomSize());
+            case 4:
+                return new IsoscelesTrapezoid(color, getRandomSize(),
+                        getRandomSize(), getRandomSize());
+            default:
+                return getDefaultFigure();

```

Comment:
```suggestion
            case 4:
            default:
                return new IsoscelesTrapezoid(color, getRandomSize(),
                        getRandomSize(), getRandomSize());
```
default case here would mean a programmer's mistake. It shouldn't ever happen